build.sbt

```diff
@@ -56,7 +56,9 @@ lazy val buildSettings = Seq(
 lazy val dockerSettings = Seq(
   Docker / maintainer := "Snowplow Analytics Ltd. <support@snowplowanalytics.com>",
   dockerBaseImage := "snowplow/base-debian:0.2.2",
+  Docker / daemonUserUid := None, 
   Docker / daemonUser := "snowplow",
+  Docker / defaultLinuxInstallLocation := "/snowplow",
```

Comment:
output binaries based on docs and docker base image.

Reply:
This is an interesting suggestion.... because as we have been migrating snowplow apps to use a different base image, we have been setting the install location to `/opt/snowplow`.  For example [here in the elasticsearch loader](https://github.com/snowplow/snowplow-elasticsearch-loader/blob/1.0.1/project/BuildSettings.scala#L54) and [here in RDB loader](https://github.com/snowplow/snowplow-rdb-loader/blob/1.2.0/project/BuildSettings.scala#L167).

Do you see any problem if we continue with this trend to put it in `/opt/docker`?  Does this conflict with any documentation you have read?  I might need to go and amend the docs, or provide an upgrade guide.

Reply:
Actually I think /opt/snowplow is probably a good convention overall. The docs I am referencing are [here](https://docs.snowplowanalytics.com/docs/pipeline-components-and-applications/stream-collector/setup/) although in looking at those docs again it is really just referencing the config file more than the binaries. Still always good to have the binaries in a consistent / known location. Also it would probably actually make sense to have the config files in a /opt/snowplow/config folder since you cannot mount a single file using K8S easily and mounting a directory removes the previous contents of said directory.

Reply:
I think the docs are consistent with the new convention:

- The application code is in `/opt/snowplow`
- The docs suggest to mount configuration files into `/snowplow`

The `/snowplow` directory does not exist in the base image, so if you mount an entire directory to that location then there is no danger of removing previous contents.